src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/prisonapi/PrisonApiClientTest.kt

```diff
@@ -59,15 +61,6 @@ class PrisonApiClientTest {
     appointments.map { it.locationId } containsExactlyInAnyOrder setOf(2000, 3000)
   }
 
-  @Test
```

Comment:
Valid test still?

Reply:
Same here. No, the API now returns all appointment types. We then filter them out in due course in the code where used.